src/Kokkos_ArithTraits.hpp

```diff
@@ -427,6 +427,21 @@ class ArithTraits {
   //! The one value of T; the multiplicative identity.
   static KOKKOS_FORCEINLINE_FUNCTION T one ();
 
+  /// \brief True if this type T is capable of representing the
+  /// positive infinity as a distinct special value, as with
+  /// std::numeric_limits<T>::has_infinity.
+  static constexpr bool hasInfinity = false;
```

Comment:
It would be better for this to be spelled `has_infinity`, for compatibility with `std::numeric_limits`: https://en.cppreference.com/w/cpp/types/numeric_limits

Reply:
Will change.  I had assumed it was the style to follow when looking at `isInf()`.

Reply:
@dalg24 Thanks! :D